### ISSUE TITLE
Peer review of color contrast research

### ISSUE THREAD

**Issue by mfairchild365** [w3c/wcag3#307]
The work, research, and changes to the color contrast formula is an amazing step in the right direction from what I can tell.

However, I'm not a color scientist and I honestly don't understand the science behind it. I can't find anything regarding whether or not the research has been peer reviewed by other color scientists. This raises further questions and concerns for me because this sort of change could have vast ramifications into the future.

Questions that I'd like answered:

* Has the research and outcomes been peer reviewed? If yes, by who and what are their credentials? If no, when will it be peer reviewed?
* Was the research tested with actual humans? If so, what was the makeup of the sample, sample size, and research methodology? If no, will this happen?
* Is the research reproducible? 

Again, I'm pleased to see progress here, but I'd feel much more comfortable if I could answer those questions.

**Comment by GreggVan** [comment]
**FIRST  -- call it CONTRAST**  

Hey -- can we stop calling this color contrast -- and just call it Contrast  --   (it is really luminance contrast  but Contrast is good enough)

We are not talking   "colors that contrast with each other".   Red and Green  have color contrast - but if they have the same luminance then people whot red-green colorblindness will see no difference. 

So what we need is  **Luminance Contrast** ---  or just **Contrast** will do but "color contrast" is confusing, misleading, and not what we are talking about.   Not helpful to those who are struggling to understand.  

**SECOND -- GOOD to reexamine**

As the originator of the old contrast measures -   I  welcome the new work and formulae.   The old ones were based on the different display technologies used at the time -- and the research and experts available at the time.  We are now using new diplay technologies, different color spaces, and additional research.    There were (and will always need to be) compromises in coming up with a final rule/equation/threshold.   But we should totally review it at this point in time


**THIRD -- Remember it is Low vision and colorblindness**

The last contrast measures were mean to both make color-coded information accessible to those with color vision differences. 

Color alone makes it hard for color coded information.  But with regard to contrast -- to facilitate reading -- both color vision differences and low vision can make it hard to read text that does not contrast with its background.


Best

**Comment by Myndex** [comment]
> _**FIRST -- call it CONTRAST**_
> _Hey -- can we stop calling this color contrast -- and just call it Contrast -- (it is really luminance contrast but Contrast is good enough)_

The term we use at [Inclusive Reading Technologies (IRT)](https://readtech.org/ARC/#visual-contrast-text) is "Visual Contrast", not color contrast. We also use the term "lightness contrast" in an effort to be plain language, and avoid less common terms such as luminance. 

Though also, contrast is a _perception_, and luminance is a _measure_. Lightness is a perception, and context sensitive. Therefore, predicting the perception of lightness contrast is the ultimate goal for a reasonable and useable guideline. 

A simple contrast of two luminance values is not a useful predictor of contrast perception, and importantly, it will not be perceptually uniform in any way. A lack of perceptual uniformity prevents a useful or practical guideline.

The science and models of perception are well understood, and easily reduced to a practical and useable guideline. That is the underlying, primary objective of APCA. 




> _We are not talking "colors that contrast with each other". Red and Green have color contrast - but if they have the same luminance then people whot red-green colorblindness will see no difference._

And red has substantially less luminance than green, and those with protanopia have a significant luminance deficit when it comes to red primaries on displays. sRGB has the least negative impact as far as the red primary, among the various common colorspaces. But a REC2020/2100 color space specifies a red that is at such a long wavelength that it is essentially invisible to protanopia.

sRGB is the best available "accommodating color space", though there are others in development (SACAM). As for luminance, there is a difference in needed calculation for the lightest of two colors especially for the wider gamut color spaces, and this is part of the next iteration, APCA25. 

The issue with the red-primary luminance deficit for protanopia is not considered in the WCAG&nbsp;2 contrast math.



> _So what we need is **Luminance Contrast** --- or just **Contrast** will do but "color contrast" is confusing, misleading, and not what we are talking about. Not helpful to those who are struggling to understand._

I've been saying this for six years here and elsewhere.

But, I have also been pointing out that the primary driver of visual contrast is the spatial characteristics—the line thickness or font weight. This is well understood, and demonstrated relative to the human contrast sensitivity curve, here: [**Empirical References**](https://readtech.org/ARC/tests/visual-readability-contrast/?tn=resources#apca-basis)



> _**SECOND -- GOOD to reexamine**_
> 
> _As the originator of the old contrast measures - I welcome the new work and formulae. The old ones were based on the different display technologies used at the time -- and the research and experts available at the time. We are now using new diplay technologies, different color spaces, and additional research. There were (and will always need to be) compromises in coming up with a final rule/equation/threshold. But we should totally review it at this point in time_

The studies in low vision readability from Legge, Whittaker, Lovie-Kitchin, et alia, were largely in the 1980s and 1990s, though certainly continuing today. The modern standard visual appearance model, CIECAM02 was 2002. Though again, there are other more recent models and continuing research in this field.

**The APCA and the APC-Readability Criterion is derived from this corpus of peer-reviewed research in readability for low vision.**

I recognize that this has not been appropriately acknowledged, despite the many bibliographies and references we have presented. This points to some institutional politics here that I am not going to get into nor arbitrate. 

It is for this reason that we founded IRT as a non-profit, to develop these standards in a science-supportive environment. We do hope that this organization chooses to adopt science-based guidelines relating to perception.



> _**THIRD -- Remember it is contrast to help those with LOW VISION - _AND_ - those with COLORBLINDNESS**_

I sense this is an unpopular opinion of mine, but contrast is needed to enable readability for 100% of human visual users. **100% of users have user needs**. User needs is a spectrum that does not start or stop with certain disabilities or visual deficits. And importantly, visual user needs are often in sharp conflict, pointing to personalization as the only true, overarching accommodation guideline.


### *READING:* 
Requires good lightness (luminance) contrast. It is only the luminance signal that is directed to the VWFA and lexical processing. This is true for all users. Hue/chroma contrast is separate, and while Legge et al. showed that standard vision can under the correct conditions read well with only color contrast, the stimuli was very large—hue/chroma signal is at less than a third the resolution of luminance.

_**LUMINANCE IS KING in the world of reading**_. This is true for all visual users. But _contrast sensitivity_ is directly and **more affected by spatial considerations**. Lightness contrast is very context dependent on the surround. Any useable guideline needs to take all these into consideration.

- Spatial (weight/thickness) directly affects contrast sensitivity.
  - Associated: white space, padding, line length, etc.
- Lightness contrast (corrects for sensitivity deficits) due to spatial or other.
  - Associated: surround and environment directly affects, as does spatial (above).
- Size in visual angle (directly accommodates acuity).
  - This is the one factor that **requires personalization**, that is, user adjustable font sizing, and not by some overall percentage, but relative, and leveraging the technology of dynamic text such as reflow.

Attempting to isolate these _fundamental_ factors will not create a useful guideline. 


### *LOW VISION:* 
Which kind? If we are talking about _acuity_, increasing lightness contrast is not the best correction, increasing the font size generally is the best correction. Increasing lightness contrast may or may not be helpful, and in some cases, increasing can have a negative effect. Similarly, increasing size beyond a point also has a negative effect on readability.

If we are talking about low vision due to occlusion, or contrast sensitivity issues, then yes increasing lightness contrast is the solution. Some with glaucoma for instance may have excellent acuity but severe contrast sensitivity deficits.


### *COLOR INSENSITIVE VISION:* 
It is well understood that as far as luminance contrast is concerned, not to mention acuity, that those with Deutan/Protan types of CVD have better contrast sensitivity than standard vision, and better visual function overall.

For our purposes there is one key exception, I mentioned above, and that is protanopia, as the red primary in a display becomes a diminishing contributor to their luminance and lightness perception. 

The rare rod-only achromatopsia and blue cone monochromacy types, fall into an AT category, as they are also afflicted with very low visual acuity and photophobia. They require assistive technology, at least insofar as maintaining low retinal illumination and very large letter forms. 



> _The last contrast measures were mean to both make color-coded information accessible to those with color vision differences._

Hue variations are covered by 1.4.1, but 1.4.3 does nothing of assistance here. Deutan/protan in particular have better contrast sensitivity—and the 1.4.3 contrast math does not accommodate the deficit caused by the red primary at all. 

The basic (current public) APCA does, mainly because it correctly predicts contrast for dark colors including reds. But as I've stated, and as shown in the published [patent](https://patents.google.com/patent/US20230316596A1/en?oq=20230316596) app, there is a better way to accommodate protans. This will be released this year in **APCA25**.


> Color alone makes it hard for color coded information. But with regard to contrast -- to facilitate reading -- both color vision differences and low vision can make it hard to read text that does not contrast with its background.

Color, as in hue, should never be "alone" in any good design. Far before the web existed, the mantra for print was always _"make it right in black and white"_, meaning that a greyscale version (aka luminance only) of the design must work on its own, without regard to color as in hue. This was (at least back in my day) taught in art and design, and certainly applies here.



Thank you for reading,


Andy


_Andrew Somers_    
_Director of Research_     
_Inclusive Reading Technologies, Inc._    
[ReadTech.org](https://readtech.org/ARC/)     

## The Revolution Will Be Readable
